sklearn/model_selection/_validation.py

```diff
@@ -115,9 +115,12 @@ def cross_validate(estimator, X, y=None, groups=None, scoring=None, cv=None,
             - A string, giving an expression as a function of n_jobs,
               as in '2*n_jobs'
 
-    return_train_score : boolean, default True
+    return_train_score : boolean, optional
         Whether to include train scores in the return dict if ``scoring`` is
         of multimetric type.
+        Current default is ``'warn'``, which behaves as ``False`` in addition
```

Comment:
This should be: "Current default is `'warn'`, which behaves as **True**", right? Otherwise, there is no point in raising a `FutureWarning` is the behavior does not change.

Reply:
I would also add the following note to this docstring entry:

Computing training scores is used to get insights on how different parameters settings impact the overfitting/underfitting trade-off. However computing the scores on the training set can be computationally expensive and is not strictly required evaluate the generalization performance.